### PR TITLE
Njsk 1.09 rollup;  fix constraints and new examples

### PR DESCRIPTION
**Pull request by nick-knowles**
This is intended to be a corrected baseline before  adding the 1.1 review comment enhancements.
It rolls up 
  (a) Various missing minor fixes from the 1.09  verioin
  (b) Other interim fixes from  no, nl etc. 
  (c) Some new examples
  (d) Some minor documentation corrections

### CODE REVIEW

File: xsd/netex_part_1/part1_tacticalPlanning/netex_fareZone_version.xsd [hunk 1]
```diff
@@ -496,6 +502,11 @@ Rail transport, Roads and Road transport
 									<xsd:documentation>FARE SECTIONS in FARE ZONE.</xsd:documentation>
 								</xsd:annotation>
 							</xsd:element>
+							<xsd:element name="contains" type="tariffZoneRefs_RelStructure" minOccurs="0">
+								<xsd:annotation>
+									<xsd:documentation>Other fare zone that are wholly  included in the  FARE ZONE.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_tacticalPlanning/netex_fareZone_version.xsd]: two extra spaces

File: xsd/netex_part_1/part1_tacticalPlanning/netex_fareZone_version.xsd [hunk 2]
```diff
@@ -538,6 +549,11 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>FARE SECTIONS in FARE ZONE.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="contains" type="tariffZoneRefs_RelStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Other fare zone that are wholly  included in the  FARE ZONE.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_tacticalPlanning/netex_fareZone_version.xsd]: two extra spaces

File: xsd/netex_part_1/part1_tacticalPlanning/netex_fareZone_version.xsd [hunk 3]
```diff
@@ -19,9 +19,15 @@
...
 				</Date>
-					<Date>
+				<Date>
 					<Modified>2018-10-01</Modified>NK: Correct  substitution group on FarePointInPattern  to PointInLinkSequence
 				</Date>
+				<Date>
+					<Modified>2019-10-01</Modified>UK-20 Add contains relationship
+				</Date>
+				<Date>
+					<Modified>20109-02-25</Modified>UK-05 Fix: Introduce a dummy Tariffzone  so that Fare zone can be a specialisation of TariffZone. Workaround to XMLSPY issue.
```
Comment (skinkie) [inline xsd/netex_part_1/part1_tacticalPlanning/netex_fareZone_version.xsd]: extra space

File: xsd/netex_part_1/part1_tacticalPlanning/netex_fareZone_support.xsd [hunk 4]
```diff
@@ -23,10 +23,13 @@
 				</Date>
 				<Date>
 					<Modified>2017-08-10 </Modified>Make FareSection a type of GeneralSection not CommonSection	 
-				</Date> 
+				</Date>
 				<Date>
 					<Modified>2018-10-01</Modified>NK: Correct  substitution group on FarePointInPattern  to PointInLinkSequence
 				</Date>
+			<Date>
+					<Modified>20109-02-25</Modified>UK-05 Fix: Introduce a dummy TariffZoneRref_ so that FareZone can be a specialisation of TariffZone. Workaround to XMLSPY sg  issue.
```
Comment (skinkie) [inline xsd/netex_part_1/part1_tacticalPlanning/netex_fareZone_support.xsd]: what kind of issue? extra space.

File: xsd/netex_framework/netex_genericFramework/netex_zone_support.xsd [hunk 5]
```diff
@@ -17,6 +17,9 @@
 				<Date>
 					<Modified>2010-11-05</Modified>
 				</Date>
+				<Date>
+					<Modified>20109-02-25</Modified>UK-05 Fix: Introduce a dummy TariffzoneRef so that Fare zone can be a specialisation of TariffZone. Workaround to XMLSPY issue.
```
Comment (skinkie) [inline xsd/netex_framework/netex_genericFramework/netex_zone_support.xsd]: 20109

**Review by ue71603 — APPROVED** [review]

**Review by ue71603 — CHANGES_REQUESTED** [review]
This is a very old PR and also we haddd Nddf

**Comment by ue71603** [comment]
I think we should close this PR. I guess most was done in new_modes anyhow. @nick-knowles  @Aurige  @skinkie : What do you think?

**Comment by skinkie** [comment]
@ue71603 agreed.

**Pull request closed without merging**